text/0000-deprecate-http-tx.md

```diff
@@ -0,0 +1,121 @@
+
+# Motivation
+[motivation]: #motivation
+
+Previously, Grin slates could be exchanged over HTTP(S) as a convenient method to take the required steps to exchange slate data between wallets to build a transaction. This method was commonly adopted by exchanges and services who already had infrastructure in place to configure wallets to listen for incoming HTTPS requests.
+
+Unfortunately, while convenient, this method of exchange requires the sender and receiver to reveal their IP addresses to route the network packets. This is bad for user privacy because without taking extra obfuscating steps, an IP address can often be traced to an individual.
+
+While steps can be taken to obfuscate a users true IP (like using a VPN), it is undesirable for a privacy-preserving technology like Grin to rely on this method for conventional use. It does not seem natural for users that are concerned about their privacy enough to use a technology like Grin to at the same time accept that their IP address is revealed when exchanging Grin with many services.
+
+Deprecating HTTP(S) provides a few benefits:
+  - User experience is improved by not having to manage firewall settings like port forwarding to accept Grin transactions from most users
+  - Wallet developer experience is improved with the need to support fewer transaction methods
+  - Users will no longer be encouraged to engage in transaction methods that potentially violate privacy during the transaction building process
+  - The ecosystem can converge around a universally accepted transaction flow that is privacy preserving
+
+The overall expected outcome is a privacy improvement for users that may otherwise have to reveal their IP address to send or receive Grin.
+
+# Community-level explanation
+[community-level-explanation]: #community-level-explanation
+
+Deprecating HTTP(S) means that the old method of sending and listening for transactions over traditional HTTP(S) is no longer supported to help preserve privacy for Grin users who may not be aware of the risk to privacy using HTTP(S) poses.
+
+This reduces the overall number of transaction transport methods that need to be supported but requires that all services and wallets that previously only accepted/sent HTTP(S) transactions to upgrade at least to the version this RFC is implemented in to support their users.
+
+For the average user this means that to send or receive Grin they can use the [Slatepack](https://github.com/mimblewimble/grin-rfcs/blob/master/text/0015-slatepack.md) method. It is no longer possible to send and receive Grin via the HTTP(S) method without custom configuration.
```

Comment:
what is meant with "without custom configuration"?

Reply:
Technically it would be possible with some custom support to continue to use http(s) to relay transactions, but it would not be possible with the standard implementation. Maybe "configuration" is the wrong word to use because it is would not be configurable at all with the standard implementation.